### DISCUSSION TITLE
History of attempts/experiements

### DISCUSSION THREAD

**Discussion by gautambak** [keras-team/autokeras#1306] [General]
Hi there,

This is a cross-post between a question posted on the issues tab. 

I was wondering how I can get a list of experiments (network architecture) and results? I wanted to use autokeras to learn about the performance of different architectures over others.

Also, is there any other approaches you suggest to leverage autokeras as a learning tool?

**Comment by haifeng-jin** [comment]
Currently, this feature is not explicitly supported. You may try to do `your_auto_model.tuner.get_best_models(10)` to get the best 10 models.

**Reply by gautambak** [reply]
You interact with more students than me so you know best, but I would think this is a good feature for people to learn the impact of various architectures on their data. I know personally as I'm learning knowing the impact of X layers or Y activation function was mostly unknown to me, seeing the results of a wide range(including non-performing ones) would have been helpful.

I thought I'd add that feedback
